x-pack/plugins/reporting/server/core.ts

```diff
@@ -108,6 +108,7 @@ export class ReportingCore {
 
     this.getContract = () => ({
       usesUiCapabilities: () => syncConfig.roles.enabled === false,
+      registerExportType: this.exportTypesRegistry.register,
```

Comment:
The way the `register` function is pulled away from the ExportTypesRegistry class makes `this` stop working in the ExportTypesRegistry.register function. I believe the code needs to arranged this way so that ExportTypesRegistry keeps its `this` context:

```suggestion
      registerExportType: (id) => this.exportTypesRegistry.register(id),
```

Reply:
Wooo got Kibana running locally in [a70912d](https://github.com/elastic/kibana/pull/155081/commits/a70912de4ab0bd7fcdd63b1116c8905de3264aa8)